nerdlandbot/translations/Translations.csv

```diff
@@ -150,6 +150,24 @@
 "empty_poncho_list","No Poncho pictures were found...","Er werden geen foto's van Poncho gevonden..."
 "poncho_pic_help","Post a random Poncho picture.","Post een willekeurige foto van Poncho."
 "poncho_hond","DOG DOG DOG","HOND HOND HOND"
+"add_mod_channel_usage","<channel>","<kanaal>"
+"add_mod_channel_brief","**admin-only** \n \u2003 Set a channel for moderator messages.","**admin-functie** \n \u2003 Stel een kanaal in voor moderator berichten"
```

Comment:
In the english version it ends with a dot, maybe add this to the Dutch version too?